sklearn/feature_selection/sequential_feature_selector.py

```diff
@@ -0,0 +1,332 @@
+"""
+Sequential feature selection
+
+"""
+
+# Author: Sebastian Raschka <se.raschka@gmail.com>
+#
+# License: BSD 3 clause
+
+import numpy as np
+from itertools import combinations
```

Comment:
we conventionally put stdlib in a first section, then external deps, then local deps

Reply:
makes sense, fixed that